test/browser/spider.js

```diff
@@ -19,6 +25,20 @@ function findSrcSetURLs(resourceText, baseURL) {
   return urls;
 }
 
+function findOpenGraphURLs(resourceText, baseURL) {
+  var urls = [];
+
+  resourceText.replace(/\sproperty="og:image"\s?content\s?=\s?['"]([^"']+)/ig, function() {
+    arguments[1].split(',').forEach(function(resource) {
```

Comment:
Is there a way to specify each argument(by giving it a more descriptive name) in the function's signature so we can grab the argument we want directly instead of using `arguments[1]`?  I don't use the `replace` method that often and after reading [MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter) it's still not immediately clear to me what `arguments[1]` refers to.  :sob: